lib/error.js

```diff
@@ -4,14 +4,12 @@ var util = require('util'),
     Controller = require('hmpo-form-controller');
 
 function getArgs(type, args) {
-    if (type === 'maxlength') {
-        return { maxlength: args[0] };
-    } else if (type === 'minlength') {
-        return { minlength: args[0] };
-    } else if (type === 'exactlength') {
-        return { exactlength: args[0] };
-    } else if (type === 'past') {
+    if (type === 'past') {
         return { age: args.join(' ') };
+    } else if (args !== undefined) {
```

Comment:
My preference would be to put some validation in the `getArgs` function; something to check `args` but always returning at least an empty object literal.